README.md

```diff
@@ -159,11 +157,11 @@ AIConfig helps unwind complexity by separating prompts, model parameters, and mo
 
 ## Features
 
-- **Prompts as Configs**: a [standardized JSON format](https://aiconfig.lastmileai.dev/docs/overview/ai-config-format) to store prompt chains and model settings in source control.
+- **Prompts as Configs**: [standardized JSON format](https://aiconfig.lastmileai.dev/docs/overview/ai-config-format) to store prompt and model settings in source control.
 - **Editor for Prompt Chains**: Prototype and iterate on your prompt chains and model settings in [AI Workbooks](https://lastmileai.dev/workbooks/clooqs3p200kkpe53u6n2rhr9).
-- **Model-agnostic SDK**: Access the latest models from OpenAI, Google, Meta, and HuggingFace - easily extensible to work with any generative AI model.
-- **Multimodal Capabilities**: Build multimodal AI apps that chain text, image, and audio models together.
-- **Improved Collaboration**: Different people can be responsible for prompt engineering and app development. AIConfig makes it easy to work together given the separation of concerns between generative AI settings and app code.
+- **Model-agnostic SDK**: Access the latest models from OpenAI, Google, Meta, and HuggingFace - easily support your own model.
```

Comment:
nit, I kind of like the previous wording better. My thought reading this is that it only supports models from those mentioned or ones that I host somewhere